tests/tst_stringsorter.qml

```diff
@@ -75,12 +75,11 @@ Item {
 
             verify(testModel.count === sorter.expectedValues.length,
                    "Expected count " + sorter.expectedValues.length + ", actual count: " + testModel.count);
-            for (var i = 0; i < testModel.count; i++)
-            {
-                var modelValue = testModel.get(i, sorter.roleName);
-                verify(modelValue === sorter.expectedValues[i],
-                       "Expected testModel value " + sorter.expectedValues[i] + ", actual: " + modelValue);
+            let actualValues = [];
+            for (var i = 0; i < testModel.count; i++) {
+                actualValues.push(testModel.get(i, sorter.roleName));
             }
+            compare(actualValues, sorter.expectedValues);
```

Comment:
There is one failing test that I don't know how to fix:

```
FAIL!  : SortFilterProxyModel::StringSorterTests::test_stringSorters(doNotIgnorePunctuation) Compared values are not the same
   Actual   (): [a-a,aa,b-b,b-c,b.c,bc]
   Expected (): [aa,a-a,b.c,b-b,bc,b-c]
   Loc: [/Users/mitch/dev/SortFilterProxyModel/tests/tst_stringsorter.qml(82)]

```

Reply:
Yeah this test is finnicky, it depends on ICU. I think it passes on Windows

Reply:
I blacklisted it on macOS.